src/Car.java

```diff
@@ -34,17 +35,23 @@ public Car(String model, String owner) {
 
     public double valueAfter(int years, double milesDriven) {
         //value * depreciationMilesFactor * milesDriven
-        double depreciatedMilesValue = value*depreciationMilesFactor*milesDriven;
+        double depreciatedMilesValue = value * depreciationMilesFactor * milesDriven;
         //value x depreciationTimeFactor * years
-        double depreciatedTimeValue = value*depreciationTimeFactor*years;
+        double depreciatedTimeValue = value * depreciationTimeFactor * years;
         //subtract both top from value initial.
         double depreciatedValue = value - depreciatedMilesValue - depreciatedTimeValue;
         return depreciatedValue;
     }
 
-    public double costOfGas(float milesDriven) {
+    public double costOfGas(double milesDriven) {
         //costOfGas = milesDriven / milesPerGallon * gasPrice
-        double overallGasCost = milesDriven/milesPerGallon*2;
+        double overallGasCost = milesDriven / milesPerGallon * 2;
         return overallGasCost;
     }
-}
\ No newline at end of file
+
+    public double lifetimeGasUsed(double milesDriven, double milesPerGallon) {
+        //how many int miles drive / mile per gallon return
+        double lifetimeGasUsed = milesDriven / milesPerGallon;
+        return lifetimeGasUsed;
```

Comment:
this returns a number called lifetimeGasUsed, but there is also a field on the Car called lifetimeGasUsed

Whoever calls this function needs to be like "lanceCar.lifetimeGasUsed(10000)" so that means the variable shouldn't exist

remove variable called lifetimeGasUsed